Test/Integration/Product/ProductsIndexingTest.php

```diff
@@ -27,17 +53,60 @@ public function testIncludingOutOfStock()
 
         $this->setOneProductOutOfStock();
 
+        $this->processTest($this->productsIndexer, 'products', $this->assertValues->productsOutOfStockCount);
+    }
+
+    public function testDefaultIndexableAttributes()
+    {
+        $empty = $this->getSerializer()->serialize([]);
+
+        $this->setConfig('algoliasearch_products/products/product_additional_attributes', $empty);
+        $this->setConfig('algoliasearch_instant/instant_facets/facets', $empty);
+        $this->setConfig('algoliasearch_instant/instant_sorts/sorts', $empty);
+        $this->setConfig('algoliasearch_products/products/custom_ranking_product_attributes', $empty);
+
         /** @var Product $indexer */
         $indexer = $this->getObjectManager()->create(Product::class);
```

Comment:
Did we want to use the new member variable here `$this->productsIndexer` or do we want to create a new instance?

Reply:
Yes totally, I'm replacing all the instances, this one will be part of it... in the next PR as well :)